Build failure on latest 2.5 branch

### Bug Description

On Arch I'm seeing the following build failure if I try to build the `2.5` branch (2b7e5050fe66). Normally I wouldn't report something like this, since it's almost certainly me doing something wrong (I know absolutely nothing about cmake or C++), but there was a thread on the forums asking for people to test 2.5.1 so I thought I'd report it. The `main` branch (706cb60c3cac) appears to build fine.

```
$ cmake --build build/ --parallel "6" --target mixxx

…

In file included from /home/sam/Projects/mixxx/src/qml/qmlwaveformoverview.cpp:4:
/home/sam/Projects/mixxx/build/mixxx-qml-lib_autogen/include/moc_qmlwaveformoverview.cpp: In static member function ‘static void mixxx::qml::QmlWaveformOverview::qt_static_metacall(QObject*, QMetaObject::Call, int, void**)’:
/home/sam/Projects/mixxx/build/mixxx-qml-lib_autogen/include/moc_qmlwaveformoverview.cpp:237:43: error: no matching function for call to ‘mixxx::qml::QmlWaveformOverview::rendererChanged()’
  237 |                 Q_EMIT _t->rendererChanged();
      |                        ~~~~~~~~~~~~~~~~~~~^~
In file included from /home/sam/Projects/mixxx/src/qml/qmlwaveformoverview.cpp:1:
/home/sam/Projects/mixxx/src/qml/qmlwaveformoverview.h:61:10: note: candidate: ‘void mixxx::qml::QmlWaveformOverview::rendererChanged(Renderer)’
   61 |     void rendererChanged(mixxx::qml::QmlWaveformOverview::Renderer renderer);
      |          ^~~~~~~~~~~~~~~
/home/sam/Projects/mixxx/src/qml/qmlwaveformoverview.h:61:10: note:   candidate expects 1 argument, 0 provided
make[3]: *** [CMakeFiles/mixxx-qml-lib.dir/build.make:502: CMakeFiles/mixxx-qml-lib.dir/src/qml/qmlwaveformoverview.cpp.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:1654: CMakeFiles/mixxx-qml-lib.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:355: CMakeFiles/mixxx.dir/rule] Error 2
make: *** [Makefile:212: mixxx] Error 2
```

### Version

2.5.1 (2b7e5050fe66)

### OS

Arch Linux 6.14.2

--- a/src/qml/qmlwaveformoverview.h
+++ b/src/qml/qmlwaveformoverview.h
@@ -58,7 +58,11 @@ class QmlWaveformOverview : public QQuickPaintedItem {
   signals:
     void playerChanged();
     void channelsChanged(mixxx::qml::QmlWaveformOverview::Channels channels);
+#if QT_VERSION >= QT_VERSION_CHECK(6, 9, 0)
+    void rendererChanged(Renderer renderer);
+#else
     void rendererChanged(mixxx::qml::QmlWaveformOverview::Renderer renderer);
+#endif
     void colorHighChanged(const QColor& color);
     void colorMidChanged(const QColor& color);
     void colorLowChanged(const QColor& color);

